### PR TITLE
Update master branch with original repo

### PR DESCRIPTION
**Pull request by fluiddot**
This PR updates the forked repo from the origin preserving the fix @sbarrio did on https://github.com/lingokids/react-native-audio-toolkit/pull/3.

### CODE REVIEW

File: ios/ReactNativeAudioToolkit/ReactNativeAudioToolkit/AudioPlayer.m [hunk 1]
```diff
@@ -222,7 +235,7 @@ - (NSURL *)findUrlForPath:(NSString *)path {
         if (version >= 10.0) {
             player.automaticallyWaitsToMinimizeStalling = false;
```
Comment (sbarrio) [inline ios/ReactNativeAudioToolkit/ReactNativeAudioToolkit/AudioPlayer.m:236]: I was hoping they would have fixed our original reason for forking it, but seems like they don't are about iOS9 anymore...

Good news it that we can go back to the repo dependency once we git rid of iOS9 and update to RN 0.63.0 🙂

**Review by sbarrio — APPROVED** [review]
Thanks!, everything looks good to me, but I'll still run a quick set of tests to see if anything broke, as I've made some fixes since we integrated it and maybe some of them don't play nice with this updated version.

**Comment by fluiddot** [comment]
> Thanks!, everything looks good to me, but I'll still run a quick set of tests to see if anything broke, as I've made some fixes since we integrated it and maybe some of them don't play nice with this updated version.

Sure, this PR will only update `master` but no the app, we need to create another PR in the app repo changing the the reference to this repo.

**Comment by sbarrio** [comment]
Cool, I'll do that and run the tests there then :)

**Comment by fluiddot** [comment]
> Cool, I'll do that and run the tests there then :)

The SHA to use is `87913e7c1e995658e0ef3c5da218a92a9a13b50a` (https://github.com/lingokids/react-native-audio-toolkit/commit/87913e7c1e995658e0ef3c5da218a92a9a13b50a).

**Pull request merged**